src/sequence.ts

```diff
@@ -0,0 +1,42 @@
+interface SequenceTrackerOptions {
+  onReset?: () => void
+}
+
+export default class SequenceTracker {
+  static readonly CHORD_TIMEOUT = 1500
+
+  private _path: readonly string[] = []
```

Comment:
A bit nit, but the underscore seems superfluous and inconsistent with the other private fields.

Reply:
I needed some way to distinguish this from the public `path` getter. Would you prefer a different naming approach? 

I did try using real private class fields (`#path`) here but ran into some compilation issues due to using an old TS version. I think it wouldn't be hard to resolve but I didn't want to get too into the weeds in this PR

Reply:
Ah didn't realize that we didn't have proper `#private` fields. Carry on then! This makes sense